babel.config.js

```diff
@@ -4,7 +4,8 @@ module.exports = {
 		['@babel/plugin-proposal-decorators', { legacy: true }],
 		'react-native-reanimated/plugin',
 		'@babel/plugin-transform-named-capturing-groups-regex',
-		['module:react-native-dotenv']
+		['module:react-native-dotenv'],
+		['transform-inline-environment-variables', { include: ['RUNNING_E2E_TESTS'] }]
```

Comment:
What's the benefit of this?
We use env var for Storybook as well while working on debug mode.